rma_category/README.rst

```diff
@@ -0,0 +1,37 @@
+==============
```

Comment:
I am afraid this will not be automatically changed so it is best to change it manually